config/imsim-user.yaml

```diff
@@ -23,10 +25,16 @@ template: /Users/walter/LSST/imSim/config/imsim-config.yaml  # Use configuration
 # Overide the the values in the template configuration with the values below.
 
 # Here are examples of local parameters you might change for your simulation.
-input.instance_catalog.file_name: /Users/walter/LSST/work/imsim-config/00466749/phosim_cat_466749.txt
+input.instance_catalog.file_name: ../tests/tiny_instcat.txt
 input.instance_catalog.sed_dir: $os.environ.get('SIMS_SED_LIBRARY_DIR') # current default
-
 input.instance_catalog.sort_mag: False
+
+# Our tiny instance catalog only uses one det, so save time not loading treerings for other dets.
+input.tree_rings.only_dets: [R22_S11]
+
+input.opsim_meta_dict:
```

Comment:
I think we should revisit how best to specify the opsim and instance info once the sky catalog stuff is working.  I consider the specification here kind of a place holder for what we will eventually want in that case.

I envision eventually we will be able to point to a full sky catalog with all the sources and a small opsim database saying where to point for this exposure, so there won't be any repetition in names.  

But for now the small instance catalog and the small opsim db aren't actually compatible (in that there are no sources found in the instance catalog where the small_opsim.db tells it to point), so I switched back to using the instance catalog for both, which at least had some galaxies to draw.

Reply:
> Also, as it is now the default is looking for "small_opsim.db". The file is in the data subdirectory. Does the code for reading this need to be modified to use `from .meta_data import data_dir` line in treerings.py ?

I don't think so.  This seems like an example file, not something that most users will use most of the time.  We'll need to provide some simple way for users to generate this kind of file I guess for their particular simulation needs.  But I don't think this exact file is going to be the thing they will use.

Reply:
Oh ... sorry... I didn't me that exact file, but to know to look in the data directory as has been set by the rest of the system.

Reply:
But if we expect users to be generating those files themselves, why would they put them in the imsim repo's data dir?  They would more likely have them in some directory specific to the simulation they were about to run, no?

Reply:
Yes, that is true. In fact, I was just puzzling over why it would be working now. I think Jim might have had a local file when he was running, but there is in facft, also one in the data directory.  I thought maybe it was to make things work by default. But I now see the entry in the .yaml file has no path.